packages/core/src/dialects/postgres/query-generator.d.ts

```diff
@@ -7,4 +7,6 @@ type PgEnumNameOptions = {
 
 export class PostgresQueryGenerator extends PostgresQueryGeneratorTypeScript {
   pgEnumName(tableName: TableName, columnName: string, options?: PgEnumNameOptions): string;
+
+  showTablesQuery(showTablesSchema?: string): string;
```

Comment:
you don't need to repeat it here, it will be inherited from query-generator

Reply:
Removed this.